modules/libunbound.jsm.js

```diff
@@ -316,7 +317,7 @@ libunboundWorker.onmessage = function(msg) {
 			/** @type {Libunbound.Exception} */
 			// @ts-ignore
 			let ex = response;
-			exception = new DKIM_InternalError(ex.message, ex.subType);
+			exception = new DKIM_TempError(ex.message, [ex.subType]);
```

Comment:
I would go with ``exception = new Error(`Error in libunboundWorker: ${ex.message}; subType: ${ex.subType}; stack: ${ex.stack}`);`` (same as in master).

Yes this is probably some kind of temporary error. But `DKIM_TempError` is supposed to take a localized string.